### DISCUSSION TITLE
Compatibility for vue 3?

### DISCUSSION THREAD

**Discussion by sennerstampontus** [mebjas/html5-qrcode#552] [Q&A]
Hello hi, 
I was looking around for a good qr/barcode scanner. And this is the best I can find. It's quick and reliable. 
I saw that there was a solution for vue 2, but i'm now in vue 3. And I was wondering if you got a solution for implementing this to a vue 3 project? I've tried working around the render on element Id thingy, because it goes to the "index.html" file, where we mount the vue app on the #app id. Is there a way to target an element inside the component? 
Thanks for your time.

**Edit**
I correct myself. I did get it to work by clicking into vue.js link in main repo and follow that guide. Although, can I style the reader with end-to-end? Or is it the "pro" option I'll go for then?

**Comment by ssherlock** [comment]
Do you have an example Vue 3 project you could share?

I am stuck with importing the JS as it throws errors.  It doesn't help that the example isn't really a Vue component (or that my knowledge in these things is a little lacking).
